apps/desktop-wallet/src/App.tsx

```diff
@@ -86,7 +86,7 @@ const App = () => {
   const _window = window as unknown as AlephiumWindow
   const electron = _window.electron
 
-  useInitializeClient()
+  useInitializeThrottledClient()
```

Comment:
Until we decide if we port this to all our projects I created a new client to avoid breaking the other apps.